### PR TITLE
update ocp to 3.7.0

### PR DESCRIPTION
**Pull request by riuvshin**
update ocp to 3.7.0

neeed for: https://github.com/eclipse/che/issues/7819

blocked by: https://github.com/eclipse/che/issues/7839

### CODE REVIEW

**Review by sleshchenko — APPROVED** [review]

File: dockerfiles/init/modules/openshift/files/scripts/ocp.sh [hunk 1]
```diff
@@ -87,11 +88,20 @@ get_tools() {
...
         wget -q -O $TOOLS_DIR/$OC_PACKAGE $OC_BINARY_DOWNLOAD_URL
         eval "$ARCH" "$TOOLS_DIR"/"$OC_PACKAGE" "$EXTRA_ARGS" &>/dev/null
         rm -rf "$TOOLS_DIR"/README.md "$TOOLS_DIR"/LICENSE "${TOOLS_DIR:-/tmp}"/"$OC_PACKAGE"
+    }
+
+    if [[ ! -f $OC_BINARY ]]; then
+        download_oc
+    else
+        if [[ $($OC_BINARY version 2> /dev/null | grep "oc v" | cut -d " " -f2 | cut -d '+' -f1 || true) != *"$OC_VERSION"* ]]; then
+            rm -rf "$OC_BINARY" "$TOOLS_DIR"/README.md "$TOOLS_DIR"/LICENSE
```
Comment (garagatyi) [inline dockerfiles/init/modules/openshift/files/scripts/ocp.sh]: Are you removing folders or files here?

File: dockerfiles/init/modules/openshift/files/scripts/ocp.sh [hunk 2]
```diff
@@ -87,11 +88,20 @@ get_tools() {
...
+        echo "download oc client $OC_VERSION"
         wget -q -O $TOOLS_DIR/$OC_PACKAGE $OC_BINARY_DOWNLOAD_URL
         eval "$ARCH" "$TOOLS_DIR"/"$OC_PACKAGE" "$EXTRA_ARGS" &>/dev/null
         rm -rf "$TOOLS_DIR"/README.md "$TOOLS_DIR"/LICENSE "${TOOLS_DIR:-/tmp}"/"$OC_PACKAGE"
+    }
+
+    if [[ ! -f $OC_BINARY ]]; then
+        download_oc
+    else
+        if [[ $($OC_BINARY version 2> /dev/null | grep "oc v" | cut -d " " -f2 | cut -d '+' -f1 || true) != *"$OC_VERSION"* ]]; then
```
Comment (garagatyi) [inline dockerfiles/init/modules/openshift/files/scripts/ocp.sh]: Can you add a comment to the file that describes what this check does and what input is expected

**Review by benoitf — APPROVED** [review]

File: dockerfiles/init/modules/openshift/files/scripts/ocp.sh (hunk 1 above)
Comment (riuvshin) [reply dockerfiles/init/modules/openshift/files/scripts/ocp.sh]: files, you think I should remove `-r`  for safety ?

File: dockerfiles/init/modules/openshift/files/scripts/ocp.sh (hunk 2 above)
Comment (riuvshin) [reply dockerfiles/init/modules/openshift/files/scripts/ocp.sh]: sure

File: dockerfiles/init/modules/openshift/files/scripts/ocp.sh (hunk 1 above)
Comment (garagatyi) [reply dockerfiles/init/modules/openshift/files/scripts/ocp.sh]: yep

**Review by garagatyi — APPROVED** [review]

File: dockerfiles/init/modules/openshift/files/scripts/ocp.sh (hunk 1 above)
Comment (riuvshin) [reply dockerfiles/init/modules/openshift/files/scripts/ocp.sh]: fixed

**Pull request merged**